# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: mozglue/baseprofiler/core/platform.cpp
# Commit: 6b399daabb18
# Full Hash: 6b399daabb187b957004361b4e53c43afc61e420
# Author: Nazım Can Altınova <canaltinova@gmail.com>
# Date: 2024-11-26 09:36:10
# Regressor Bug: 1634785
# File Overlap Count: 1
# Description:
#   Bug 1634785 - Rename BaseProfilerSharedLibraries.h to SharedLibraries.h r=mstange,profiler-reviewers
#   
#   Now that we only have one implementation of this code, we don't have to
#   differentiate it with "BaseProfiler".
#   
# ==============================================================================

diff -r 8ad3b7e6de23 -r 6b399daabb18 mozglue/baseprofiler/core/platform.cpp
--- a/mozglue/baseprofiler/core/platform.cpp	Mon Nov 25 22:09:05 2024 +0000
+++ b/mozglue/baseprofiler/core/platform.cpp	Mon Nov 25 22:09:05 2024 +0000
@@ -70,7 +70,7 @@
 #include "ProfilerBacktrace.h"
 #include "ProfileBuffer.h"
 #include "RegisteredThread.h"
-#include "BaseProfilerSharedLibraries.h"
+#include "SharedLibraries.h"
 #include "ThreadInfo.h"
 #include "VTuneProfiler.h"
 